src/js/containers/account/AccountContainer.jsx

```diff
@@ -25,10 +25,17 @@ import LoadingAccount from 'components/account/LoadingAccount';
 require('pages/account/accountPage.scss');
 
 const propTypes = {
-    currentFiscalYear: PropTypes.string,
+    latestSubmissionDate: PropTypes.string,
```

Comment:
<img width="945" alt="Screen Shot 2021-01-13 at 1 40 46 PM" src="https://user-images.githubusercontent.com/7108785/104495078-49241b80-55a5-11eb-8d85-7de8743eea6e.png">
Looks like this prop is expected to be an object and duplicated below, nested under `latestPeriod`